monitoring/remote_monitoring_demo/accelerator_demo.py

```diff
@@ -0,0 +1,326 @@
+import copy
```

Comment:
Do we need a separate demo for accelerator? Can we add this to remote monitoring demo itself?